src/main/java/com/commercetools/sync/products/ProductSyncOptionsBuilder.java

```diff
@@ -47,14 +47,14 @@ public ProductSyncOptionsBuilder syncFilter(@Nonnull final SyncFilter syncFilter
     }
 
     /**
-     * Set option that indicates whether sync process should create price channel of given key when it doesn't exists
+     * Set option that indicates whether sync process should create a price channel of given key when it doesn't exist
      * in a target project yet. If set to {@code true} sync process would try to create new price channel of given key,
```

Comment:
```suggestion
     * in a target project yet. If set to {@code true}, the sync process would try to create the new price channel of the given key,
```
https://www.ego4u.com/en/cram-up/writing/comma?10